src/plugins/shared_ux/public/components/redirect_app_links/redirect_app_links.stories.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { EuiButton } from '@elastic/eui';
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { BehaviorSubject } from 'rxjs';
+
+import { action } from '@storybook/addon-actions';
+import { RedirectAppLinks } from './redirect_app_links';
+import mdx from './redirect_app_links.mdx';
+
+export default {
+  title: 'Redirect App Links',
+  description: 'app links component that takes in an application id and navigation url.',
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+};
+
+export const Component = () => {
+  return (
+    <RedirectAppLinks
+      navigateToUrl={() => Promise.resolve()}
+      currentAppId$={new BehaviorSubject('test')}
+    >
+      <EuiButton
+        data-test-subj="homeAddData"
+        fill
+        iconType="plusInCircle"
+        onClick={action('button pressed')}
+      >
+        <FormattedMessage
```

Comment:
nit: it's just a storybook example, so no need to burden this with other unrelated components.